node/src/components/linear_chain_sync/operations.rs

```diff
@@ -438,13 +476,13 @@ pub(crate) async fn run_fast_sync_task(
                 && is_current_era(
                     &most_recent_block_header,
                     &trusted_key_block_info,
-                    &chainspec,
+                    chainspec,
```

Comment:
Under what circumstances `is_current_era` would return `true` but we would still be rocking a different version from the most recent block's header 🤔 ? Maybe we should log a warning in that case? So that we don't miss them as I think it would mean there's an error somewhere.